.travis.yml

```diff
@@ -7,22 +7,36 @@ install:
   - "pip install -r bassist/requirements.txt"
   - "pip install -e savant/."
   - "pip install -e bassist/."
+  - >
+    curl -sSL "https://get.docker.com/gpg" |
+    sudo -E apt-key add -
+  - >
+    echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" |
+    sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update
+  - docker version
+  - sudo pip install docker-compose
+  - docker-compose version
 env:
   - DB=pgsql
 service:
   - postgresql
+  - docker
 before_script:
   - psql -c 'drop database if exists travis_ci_test;' -U postgres
```

Comment:
Thes psql commands are redundant if you use postgres from your `docker-compose.yaml`